src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -299,11 +311,3 @@ def load_arguments(self, _):
         c.argument('table_name', options_list=['--name', '-n'], help="Table name")
         c.argument('throughput', type=int, help='The throughput of Table (RU/s).')
         c.argument('max_throughput', max_throughput_type)
-
-    for scope in ['sql database throughput migrate', 'sql container throughput migrate',
```

Comment:
why removing these commands? If you want to deprecate them, please add deprecate info first and remove them later for cx awareness.

Reply:
I removed the throughput migrate and analytical ttl commands because the version of the API I am using does not include them. This PR is not to merge here but I was told to do the review here since it has more checks in the CI. The goal is to merge to a separate branch in the private repo and release that package privately to our customers.

Reply:
These are not included in the version of the SDK we are using